### ISSUE TITLE
Kibbeh: Accessibility: The 'publicity' dropdown in the Create Room dialog does not have a focus state

### ISSUE THREAD

**Issue by MMK21Hub** [benawad/dogehouse#1697]
**The bug**
In the Create Room dialogue, there is no visual indicator that the 'publicity' dropdown is focused on. This hinders keyboard navigation.

**Steps to Reproduce**
1. Go to the Home screen
2. Click on "New Room"
  → ✔️ The 'Room name' text box is focused
3. Press `tab`
  → :x: The dropdown is focused, but you can't tell
4. Press `tab` again
  → ✔️ The 'Room description' text box is focused

**Expected behaviour**
There would be an outline (ideally) or some other visual indicator that the dropdown is focused on.

**What device are you on?**
- Windows 10 OS Version 2004
- Chrome 89

**Comment by MMK21Hub** [comment]
Closing due to https://www.youtube.com/watch?v=I8PkQgPiSq8
